src/components/Auth/assets/animations.scss

```diff
@@ -0,0 +1,125 @@
+:global {
+  .experiment-svg {
```

Comment:
It needs to be in a global scope as the animations need to be available throughout the whole application. Since facetec renders its components on body root, we can't pinpoint a DOM area to set these css animations.